Javascript javascript3 week2

Kositthai opened the pull request:
Submit homework JS3 Week2

cecastosic commented on javascript/javascript3/week2/main.js:
```diff
@@ -0,0 +1,90 @@
+
+// display option accroding to currency in rates
+const createOption = async () => {
+    try {
+        const response = await fetch('https://open.er-api.com/v6/latest/USD')
+            if(response.status !== 200 ){
+                throw new Error('something went wrong')
+            }
+                const data = await response.json()
+                const rates = data.rates; 
+                const keys = Object.keys(rates); 
+                createOptions(keys) 
+            } catch(error){
+                alert(`Sorry, there was a problem with the operation.`)
+                console.log(error.message)
+                return error.message; 
+            }
```
More readable option IMO would be using `fetch then catch `
Good reading https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch and https://dmitripavlutin.com/javascript-fetch-async-await/
but nice with checking the response.status

cecastosic commented on javascript/javascript3/week2/main.js:
```diff
@@ -0,0 +1,90 @@
+const inputEl = document.getElementById('inputEl'); 
+const convertFrom = document.getElementById('convertFrom'); 
+const convertTo = document.getElementById('convertTo'); 
+const btn = document.getElementById('btn');
+const displayCurrency = document.getElementById('displayCurrency');
```
Some of the variables should be declared as local variables since they are only used in specific functions

cecastosic reviewed (commented):
Nice job! I added some comments and once again I would suggest using Prettier as a code formatter.

cecastosic commented on javascript/javascript3/week2/index.html:
```diff
@@ -0,0 +1,41 @@
+                <div class="convert-section">                  
+                    <div class="convert-flex">
+                        <label>Amount</label>
+                        <input id="inputEl" class="input" placeholder="Amount" value="1">    
```
What should happen if a user enters text in the input?

Kositthai commented:
> Nice job! I added some comments and once again I would suggest using Prettier as a code formatter.

Thank you so much for your reviewed!
I have fixed some of my code referring to your suggestions 

for convert-wrapper css file, I removed width 50% and adjust .convert-section to { justify-content: center }

I have add some styles for the display-section to width: 50% and margin to auto because I want my display-section to be on the left side but in the middle of the page 

Pls let me know if I have to adjust anything else or I can merge my PR :)
